packages/fuels/vite.config.js

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  build: {
+    minify: true,
+    sourcemap: true,
+    emptyOutDir: false,
+    lib: {
+      entry: './src/index.ts',
+      name: 'browser',
+      formats: ['es'],
+      fileName: 'browser',
+    },
+  },
+  define: {
+    'process.env.NODE_ENV': JSON.stringify('production'),
```

Comment:
```suggestion
    'process.env.NODE_ENV': 'production',
```
Unnecessary use of `stringify` here?

Reply:
If I remove the `JSON.Stringfy` an error is shown:

![Screenshot from 2024-01-05 08-17-38](https://github.com/FuelLabs/fuels-ts/assets/30977845/7663ab59-8d0e-45e6-aaa7-966744d77d16)

![Screenshot from 2024-01-05 08-19-00](https://github.com/FuelLabs/fuels-ts/assets/30977845/28edfeb9-9596-409b-9253-906c4136e746)

Reply:
Can/should we go for `vite.config.ts`? (typescript)

Does it make any difference?

Reply:
With `.ts` we can have type checking and autocompletion

Reply:
Is the `JSON.stringify()` still needed when using `.ts`?

Reply:
Yes, I've just checked it :laughing:

Reply:
🤯